mkdocs.yml

```diff
@@ -51,13 +51,13 @@ pages:
     # Installation guide
     - 'Install': 'get-started/install.md'
     # Writing one's first Exonum service
-    - 'Create Service': 'get-started/create-service.md'
+    - 'Cryptocurrency Service': 'get-started/create-service.md'
```

Comment:
Maybe, it would make sense to rename this and the following pages to "Cryptocurrency Tutorial" / "Advanced Cryptocurrency Tutorial" ?